src/github.com/getlantern/flashlight/genconfig/uploadyaml.bash

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+function die() {
+  echo $*
+  exit 1
+}
+
+if [ -z "$1" ]
+then
+    die "Need to specify version string(s), e.g., '$0 default cn'"
+fi
+
+for version
```

Comment:
This is a shortcut for iterating over the arguments, so if you call `uploadyaml.bash cn default`, `version` takes the values `cn` and `default` in turn.